code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -169,15 +169,18 @@ mob/living/carbon/bullet_act(obj/item/projectile/P, def_zone)
 		O.emp_act(severity)
 	..()
 
-/mob/living/carbon/electrocute_act(shock_damage, obj/source, siemens_coeff = 1, override = 0, tesla_shock = 0)
+/mob/living/carbon/electrocute_act(shock_damage, obj/source, siemens_coeff = 1, safety = 0, override = 0, tesla_shock = 0, illusion = 0)

```

Comment:
I've tested it, and it seems to work just fine on monkeys/humans.  What do you see that's wrong, is there an edge case I've missed?